### PR TITLE
Raising a meaningful error message if iommu is not enabled in bios

### LINKED ISSUE #162: Raising a meaningful error message if iommu is not enabled in bios
When configuring a device to use the vfio driver, if IOMMU is not enabled in the bios of the machine, the resulting `SriovNetworkNodeState` will report a failure along the line of `write error: No such device` when trying to bind a device to the vfio driver.

My suggestion here is to check if iommu is enabled (by checking the content of /sys/class/iommu`, and provide a more descriptive error message, so that the admin can understand how to fix the error.

### PR DESCRIPTION
**Pull request by pliurh**
fix #162

### CODE REVIEW

**Comment by fedepaol** [comment]
Thanks!
/lgtm

File: pkg/utils/driver.go [hunk 1]
```diff
@@ -55,6 +57,11 @@ func BindDpdkDriver(pciAddr, driver string) error {
 	err = ioutil.WriteFile(bindPath, []byte(pciAddr), os.ModeAppend)
 	if err != nil {
 		glog.Errorf("BindDpdkDriver(): fail to bind driver for device %s: %s", pciAddr, err)
+		_, err := os.Readlink(filepath.Join(sysBusPciDevices, pciAddr, "iommu_group"))
```
Comment (zshi-redhat) [inline pkg/utils/driver.go:60]: when driver bind fails, there might be two reasons:
1) iommu not enabled in BIOS.  (will this result in error when setting iommu kernel argument ?)
2) iommu is enabled in BIOS, iommu kernel argument is not configured.

Shall we distinguish the two with different error messages?

Comment (fedepaol) [reply pkg/utils/driver.go:60]: Isn't 2. covered by the operator itself?

Comment (zshi-redhat) [reply pkg/utils/driver.go:60]: There was an issue reported from customer-0 that vfio bind failed but VT-d is enabled in BIOS. We didn't find the root cause as there is no clear reproducer. The fix propsed for aforementioned issue is to at: https://github.com/openshift/sriov-network-operator/pull/179. But I don't know if there are other causes.

Comment (fedepaol) [reply pkg/utils/driver.go:60]: Understood, thanks

Comment (pliurh) [reply pkg/utils/driver.go:60]: After #179, the daemon will always reboot the node if the IOMMU is not enabled in Kernel. So if case 2, the daemon will reboot the host before trying to load the driver.

**Pull request merged**